backend/src/main/java/com/benine/backend/http/PresetCreationHandler.java

```diff
@@ -40,20 +48,43 @@ public void handle(HttpExchange exchange) throws IOException {
         IPCamera ipCamera = (IPCamera)camera;
         
         Preset preset = createPreset(ipCamera);
+        int presetID = preset.getId();
         
         //Adding the new preset to the database
         getCameraController().addPreset(cameraID, preset);
+        
+        //Create corresponding image
+        createImage(preset, cameraID, presetID);
         responseSuccess(exchange);
       
       }
-    } catch (SQLException e) {
-      System.out.println(e.toString());
+    } catch (SQLException | StreamNotAvailableException e) {
       getLogger().log("Preset can not be added to the database", LogEvent.Type.CRITICAL);
```

Comment:
So you mean splitting the two exceptions and saying that or the stream wasn't availble or there was a sql exception?